<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<<<<<<< HEAD
<testsuite hostname="NYCMDLNMUHMAC.local" name="com.framework.automation.framework.homePage.VerifyloginWithValidCredentails" tests="1" failures="0" timestamp="12 Nov 2017 01:46:46 GMT" time="0.001" errors="1">
  <testcase name="verifyLoginWithValidCredentails" time="0.001" classname="com.framework.automation.framework.homePage.VerifyloginWithValidCredentails">
    <error type="org.openqa.selenium.WebDriverException" message="Timed out waiting 45 seconds for Firefox to start.
Build info: version: &#039;3.6.0&#039;, revision: &#039;6fbf3ec767&#039;, time: &#039;2017-09-27T15:28:36.4Z&#039;
System info: host: &#039;NYCMDLNMUHMAC.local&#039;, ip: &#039;2604:2000:a806:6f00:4093:e4ba:9933:6654&#039;, os.name: &#039;Mac OS X&#039;, os.arch: &#039;x86_64&#039;, os.version: &#039;10.11.4&#039;, java.version: &#039;1.8.0_60&#039;
Driver info: driver.version: FirefoxDriver">
      <![CDATA[org.openqa.selenium.WebDriverException: Timed out waiting 45 seconds for Firefox to start.
Build info: version: '3.6.0', revision: '6fbf3ec767', time: '2017-09-27T15:28:36.4Z'
System info: host: 'NYCMDLNMUHMAC.local', ip: '2604:2000:a806:6f00:4093:e4ba:9933:6654', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.11.4', java.version: '1.8.0_60'
Driver info: driver.version: FirefoxDriver
	at org.openqa.selenium.firefox.XpiDriverService.waitUntilAvailable(XpiDriverService.java:112)
	at org.openqa.selenium.firefox.XpiDriverService.start(XpiDriverService.java:97)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:79)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:586)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:217)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:140)
	at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:120)
	at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:98)
	at com.framework.automation.framework.testBase.TestBase.selectBrowser(TestBase.java:43)
	at com.framework.automation.framework.testBase.TestBase.init(TestBase.java:23)
	at com.framework.automation.framework.homePage.VerifyloginWithValidCredentails.setup(VerifyloginWithValidCredentails.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:514)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:215)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
	at org.testng.TestRunner.beforeRun(TestRunner.java:656)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
Caused by: org.openqa.selenium.net.UrlChecker$TimeoutException: Timed out waiting for [http://localhost:31913/hub/status] to be available after 45004 ms
	at org.openqa.selenium.net.UrlChecker.waitUntilAvailable(UrlChecker.java:100)
	at org.openqa.selenium.firefox.XpiDriverService.waitUntilAvailable(XpiDriverService.java:110)
	... 32 more
Caused by: java.util.concurrent.TimeoutException
	at java.util.concurrent.FutureTask.get(FutureTask.java:205)
	at com.google.common.util.concurrent.SimpleTimeLimiter.callWithTimeout(SimpleTimeLimiter.java:147)
	at org.openqa.selenium.net.UrlChecker.waitUntilAvailable(UrlChecker.java:75)
	... 33 more
]]>
    </error>
  </testcase> <!-- verifyLoginWithValidCredentails -->
=======
<testsuite hostname="kagims-MacBook-Pro.local" name="com.framework.automation.framework.homePage.VerifyloginWithValidCredentails" tests="3" failures="0" timestamp="25 Nov 2017 17:54:38 GMT" time="63.294" errors="0">
  <testcase name="vrifiedHomePageshoppingChartButtonDisplayed" time="0.040" classname="com.framework.automation.framework.homePage.VerifyloginWithValidCredentails"/>
  <testcase name="validEmailforCreateAccount" time="1.207" classname="com.framework.automation.framework.homePage.VerifyloginWithValidCredentails"/>
  <testcase name="verifyLoginWithValidCredentails" time="62.047" classname="com.framework.automation.framework.homePage.VerifyloginWithValidCredentails"/>
>>>>>>> 49ff801534aeb5c5b7564bf7e6e225a80cad317c
</testsuite> <!-- com.framework.automation.framework.homePage.VerifyloginWithValidCredentails -->
